meeting-summary/2021-10-08-基于学习的基数估计方法的可用性分析.md

```diff
@@ -0,0 +1,241 @@
+# 2021.10.08 分享纪要
+
+- 分享人: 朱道冰
+- 关键词: 查询优化、基数估计
+- Keywords: Query Optimization, Cardinality Estimation
+- 分享PPT: [基于学习的基数估计方法的可用性分析](./slides/2021-10-08-feasibility-of-learning-based-cardinality-estimation-methods.pdf)
+
+
+
+[基数估计相关方法介绍](https://github.com/CDDSCLab/Weekly-Group-Meeting-Paper-List/blob/main/meeting-summary/2021-01-08-%E5%9F%BA%E6%95%B0%E4%BC%B0%E8%AE%A1%E7%9A%84%E7%9B%B8%E5%85%B3%E6%96%B9%E6%B3%95%E4%BB%8B%E7%BB%8D.md)介绍了基数估计的概念、难点，介绍了基于统计信息的基数估计方法及基于 Query 结构进行特征编码的机器学习方法。
+
+[基于概率分布的基数估计方法](https://github.com/CDDSCLab/Weekly-Group-Meeting-Paper-List/blob/main/meeting-summary/2021-05-14-%E5%9F%BA%E4%BA%8E%E6%95%B0%E6%8D%AE%E6%A6%82%E7%8E%87%E5%88%86%E5%B8%83%E7%9A%84%E5%9F%BA%E6%95%B0%E4%BC%B0%E8%AE%A1%E6%96%B9%E6%B3%95.md)进一步介绍基于概率分布的基数估计方法。
+
+![image-methods-overview](imgs/20211008/methods-overview.png)
+
+本次分享的论文[1]对基于机器学习的基数估计方法的可行性分析。该文章使用统一的数据集和工作负载对近几年并行出现的学习方法进行横向对比实验及分析。主要关注的点有：
+
+* 现有的工作大多集中在静态的环境，没有数据更新。
+* 现有的工作缺少对模型个阶段时间开销的研究：
+  * 训练时间(training time)
+  * 预测时间(inference time)
+  * 更新时间(update time)
+* 现有的工作缺少对影响模型准确率因素的研究。
+
+## 静态环境中学习方法的可用性分析
+
+### 实验设置
+
+#### 数据集
+
+* 真实世界数据集。
+* 大小不同，列与数据分布不同。
+* 每个数据集被用于评价至少一个学习方法。
+
+![image-datasets](imgs/20211008/datasets.png)
+
+#### 负载
+
+* 同一个实验中使用相同的工作负载。
+* 支持1 ~ |列数量| 的谓词数量。
+* 同时支持等值和范围谓词。
+* OOD -> 不同列之间的谓词生成支持独立性。
+
+### 学习方法的准确率提升
+
+准确率$q_{error}$定义，越接近于1越好。
```

Comment:
Spaces between math formulas and words, or it can't be rendered correctly.
It seems that there is no rules to check math formula in markdown lint plugin.